src/main/webapp/app/exercises/shared/exercise-headers/header-exercise-page-with-details.component.html

```diff
@@ -21,6 +21,13 @@ <h4 *ngIf="!isTestRun">
                 </h4>
             </ng-template>
             <ng-template #courseMode>
+                <h4 *ngIf="notReleased()">
```

Comment:
please avoid method calls in the html template because they can severely impact the client performance (see client coding guidelines)

Reply:
Do you have a hint because adding the computation directly in the html file does unfortunately not work because I cannot use moment in the html file?

Reply:
Thank you very much. That worked :-)